src/image/Wcs.cc

```diff
@@ -217,21 +217,6 @@ void Wcs::initWcsLibFromFits(CONST_PTR(lsst::daf::base::PropertySet) const& head
         string msg = "Could not parse FITS WCS: no header cards found";
         throw LSST_EXCEPT(except::InvalidParameterError, msg);
     }
-
-    // Scamp produces PVi_xx header cards that are inconsistent with WCS Paper 2
-    // and cause WCSLib to choke.  Aggressively, rename all PV keywords to X_PV
-    for (int j=1; j<3; j++) {
-        for (int i=0; i<=99; i++) {
-            std::string key = (boost::format("PV%i_%i") % j % i).str();
-            if (!access.toRead()->exists(key)) {
-                break;
-            }
-            double val = access.toRead()->getAsDouble(key);
-            access.toWrite()->remove(key);
-            access.toWrite()->add("X_"+key, val);
-        }
-    }
-
```

Comment:
A minor formatting point, but it would be nice to reinstate one of the blank lines you've removed for the sake of the rest of the code.